### ISSUE TITLE
Github Markdown emojis support

### ISSUE THREAD

**Issue by thansk** [TriliumNext/Trilium#1011]
Add the functionality of including Github Markdown emojis.
They are all listed here, for example: https://github.com/ikatyang/emoji-cheat-sheet

**Comment by zadam** [comment]
Just to be clear, do you mean that after typing e.g. \```:grinning:` ``

**Comment by zadam** [comment]
Just to be clear, do you mean that after typing e.g. \```:grinning:` ``

**Comment by thansk** [comment]
Yes, exactly. I am coming from an app that has an edit mode and a "view" mode, so there it's not automatic, but I think here it should be.

**Comment by jgod** [comment]
👍  Emojis are great for styling headers, etc.

**Comment by eliandoran** [comment]
We've recently added support for emoji auto-completion by typing `:`.
